src/main/java/com/bisnode/opa/client/OpaClient.java

```diff
@@ -81,12 +83,20 @@ public Builder opaConfiguration(String url) {
             return this;
         }
 
+        public Builder objectMapperConfiguration(ObjectMapper objectMapper) {
```

Comment:
Most often we find builder methods named the same as fields, so here I'd suggest `objectMapper(ObjectMapper objectMapper)`